recipes/boost/all/conanfile.py

```diff
@@ -599,6 +599,12 @@ def disable_cobalt():
             if Version(self.version) == "1.86.0" and is_msvc(self):
                 setattr(self.options, "without_process", True)
 
+        if Version(self.version) >= "1.90.0":
+            # FIXME: boost.coroutine doesn't support Windows ARM64 due to missing context assembly
+            # See https://github.com/microsoft/vcpkg/issues/10473 and https://github.com/boostorg/context/issues/241
+            if self._is_windows_platform and self.settings.arch in ("armv8", "armv8.3"):
+                self.options.without_coroutine = True
```

Comment:
```suggestion
        if Version(self.version) == "1.90.0":
            # FIXME: boost.coroutine doesn't support Windows ARM64 due to missing context assembly
            # See https://github.com/boostorg/context/issues/296
            if self._is_windows_platform and "arm" in str(self.settings.arch):
                self.options.without_coroutine = True
```

Let's keep only 1.90.0, then revisit again on the next release. It was not breaking in the previous version.